### PR TITLE
Item 6932: Source - Display Source Samples, Assays, Jobs

### PR DESCRIPTION
**Pull request by XingY**

### CODE REVIEW

**Review by labkey-susanh — COMMENTED** [review]
Looks good to me, but I'll defer to Kevin on particulars here.

File: experiment/src/org/labkey/experiment/api/data/LineageHelper.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        options.setParents(true);
+        options.setChildren(false);
+        return options;
+    }
+
+    static SQLFragment createInSQL(SQLFragment fieldKeyFrag, SQLFragment lsidFrag, ExpLineageOptions options)
+    {
+        String lsidStr = lsidFrag.getRawSQL();
+
+        // FIXME remove the quote workaround from client and here when parser is able to parse parts correctly
```
Comment (labkey-kevink) [inline experiment/src/org/labkey/experiment/api/data/LineageHelper.java:63]: Do we have an issue tracking this?

File: experiment/src/org/labkey/experiment/api/data/ChildOfMethod.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package org.labkey.experiment.api.data;
+
+import org.labkey.api.data.JdbcType;
+import org.labkey.api.data.SQLFragment;
+import org.labkey.api.data.dialect.SqlDialect;
+import org.labkey.api.query.AbstractMethodInfo;
+
+public class ChildOfMethod extends AbstractMethodInfo
```
Comment (labkey-kevink) [inline experiment/src/org/labkey/experiment/api/data/ChildOfMethod.java:18]: Can you document what the expected arguments are for the method?

**Review by labkey-kevink — APPROVED** [review]
Looks good, just minor comments

**Pull request merged**